### DISCUSSION TITLE
2023-07-03

### DISCUSSION THREAD

**Discussion by MoonJeWoong** [woowacourse-teams/2023-haru-study#5] [daily_meeting]
### 지난 KPT 종합

K : 
Figjam을 이용한 미팅 및 회의 진행
칭찬할 때 따봉 날리기 Action Item 선정
회의 시 제한시간을 정하고  중간 중간 목적 및 방향 재확인 및 재설정
 
P : 
중요도가 높지 않은 사안에 너무 많은 시간을 투자한다.
계속 투표에 부치는 방식 

T :
새로운 의사결정 방식을 찾아보자
기술관련 의사결정이 아닌 중요도가 낮은 사안들은 다른 방법을 통해 빠르게 결정해보도록 하자.

### Action Item

생각 동기화 - 한 번의 회의 사이클(50분)을 시작하기 전 목표를 설정하고 회의를 시작한다.
칭찬하기 - 면전에 따봉을 날리자

### 할 일 목록

- 백엔드
    - ~~코드 컨벤션 문서화~~
    - ~~IA 그리기~~
- 프론트엔드
    - 와이어 프레임 스케치
- ~~중간과정 공유 회의~~
- 구체화된 기능 명세 최종 결정까지 마무리하고 픽스(다 안되면 내일 이어서)

### 남은 할 일

- Github milestone 설정 (?) - 아직 깃허브 프로젝트 기능이 안됨
- 문서 작성 (이슈, 디스커션)템플릿 및 작성 가이드 문서화 컨펌
    - 프로젝트, 칸반보드

### 내일 할 일
- 백엔드
  - 개발문서 마무리 (기술 스펙 및 선정 이유 정리)
- 프론트엔드
  - 와이어 프레임 스케치 마무리
- 공통
  - 새로운 의견에 대한 논의(스터디 모집 시점에 대한 논의)
  - IA 공유, 와이어 프레임 스케치 공유 후 MVP 명세 픽스
  - 솔라와 커피챗
  - 사용자 스토리 제출
  - 솔라 커피챗 전 현재 고려사항 정리해서 질문지 보내두기

**Comment by jaehee329** [comment]
### 체크인
8점 - 주말에 잘 쉬긴 했는데 너무 덥네요! 화이팅 해보죠 이번 주도

### 체크아웃
6점 - 회의 엄청 많이 했네요.. 얼른 명세 다 정해졌으면 좋겠다

### KPT
K: 목표를 명시하고 그 목표에만 맞는 회의가 잘 이뤄지는 것 같아요
P: 
T:

**Comment by woo-jk** [comment]
### 체크인

9점 - 주말에 공부를 생각보다 못했다;;;;;;

### 체크아웃

8점 - 서비스 스케치가 재밌다. 하지만 회의가 쉽게만 진행되지 않는다.

### KPT
K: 각자 할 일을 해온 뒤 설명을 하는 방식으로 회의를 진행하니 말끔하게 진행되었다.
P: 체크인, 체크아웃을 할 때 사용하는 플랫폼(깃헙, 노션, 피그잼)이 동시 다발적으로 작성되고 있어 흐름을 놓칠 때가 있다.
T: 병렬적으로 작성하지 않거나, 서기가 한 번에 작성해주면 좋을듯?

**Comment by yeopto** [comment]
### 체크인

9점 - 주말 동안 푹 쉬다와서 컨디션 좋은데 너무 더워서 -1

### 체크아웃

10점 - 오늘은 일의 진행이 수월하게 되어서 +1

### KPT

K: 오늘 새로운 방법으로 일을 진행했는데(병렬적인 업무 분담) 생각보다 잘 되어서 좋았던 것 같다.
P: 문제는 없었다.
T: 고로 Try도 없다.

**Comment by nlom0218** [comment]
### 체크인

6점 - 진짜 너무 덥다ㅠㅠㅠ 🥵 단순히 더워서 -4. 다른건 없어요😄

### 체크아웃

9점 - 강의장이 시원하다!! 과연 어떤 기능(플로우)이 좋을지 항상 의문이 든다ㅠㅠ

### KPT
K: 우리팀만의 효율적인 회의 방식을 찾아가는 느낌이 든다.
P: 여전히 새로운 아이디가, 플로우가 등장하는데 이를 어떻게 표현해야 할까?
T: 각자의 생각을 발표하는 시간 가지기

**Comment by woosung1223** [comment]
### 체크인
7점 - 햇빛이 너무 강렬하고(-1) 월요일이라(-2) 7점입니더

### 체크아웃
6점 - 한번에 IA를 잘 그렸으면 좋았을텐데, 제가 갈팡질팡한 부분이 있어서 아쉽습니다 ㅠ

### KPT
K: 그래도 어느정도는 기획의 방향성이 잡혀가는 것 같음!
P: 없을 무
T: 없을 무

**Comment by aak2075** [comment]
### 체크인

6점 - 오늘 근로 회의 진행해야 하는 부담감, 젠킨스 스크립트 실행 제대로 안됨

### 체크아웃

8점 - 근로 회의 진행도 끝나고 IA 그리는 것도 완성해서 좋음. 

### KPT

K : 무조건 다 같이 정하기 보다 가능한 것은 각자 해와서 공유하면서 회의하는 방식이 의사 결정에 도움이 되었음. 이야기가 삼천포로 빠진다 싶을 때 적절이 끊어주는 것이 좋았다.
P : 
T :

**Comment by MoonJeWoong** [comment]
### 체크인

9점 - 드디어 기초 강의 다 수강해서 기분이 좋습니다!

### 체크아웃

10점 - 그동안 서로가 동기화되지 못했던 부분에 대해 더 구체화할 수 있었던 것 같습니다!

### KPT
K :
굳이 전체 단위에서 얘기하지 않아도 될 사안에 대해서 한 사람이 좋은 방법을 준비해오면 일단 그 방식으로 시작하고 보완해나가는 방식을 채택한 것.

P :
오늘은 딱히 없었던 것 같아용.

T :
Problem이 없어서 없습니다.
